Packs/MandiantAdvantageAttackSurfaceManagement/Integrations/AttackSurfaceManagement/test_data/baseintegration-dummy.json

```diff
@@ -0,0 +1,3 @@
+{
+    "dummy": "this is a dummy response"
```

Comment:
Can delete this file?

Reply:
@chrishultin please delete this file and pull from master to re-trigger the build.